app/person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @permission = parent_permission
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+  end
+
+  def read_id
+    @id
+  end
+
+  def read_name
+    @name
+  end
+
+  def read_age
+    @age
+  end
+
+  def access_name(name)
+    @name = name
+  end
+
+  def access_age(age)
+    @age = age
+  end
```

Comment:
- [ ] I think these methods are not needed since if you want to read or access these instance variables you can call them directly for the constructor. :blush: